### PR TITLE
DEV-3766: Fix germline MVLH table in PDF

### PR DESCRIPTION
**Pull request by DFKoetsier**
Changes:
- Remove unneeded division by 100 from when the input file contained percentages instead of factors.
- There are genes that have MVLH 100% and depth 0 in the input file due to being ignored by SAGE germline entirely. Remove these genes from both the MVLH table in the PDF and the related field in the JSON.

Please feel free to add more reviewers if you want.

I couldn't think of a better way to do the test I added to OrangeAlgoTest. If you can, please let me know.

### CODE REVIEW

File: orange/src/main/java/com/hartwig/hmftools/orange/algo/OrangeAlgo.java [hunk 1]
```diff
@@ -368,38 +369,74 @@ private static EnsemblDataCache loadEnsemblDataCache(@NotNull OrangeConfig confi
...
+        Map<String, Double> mvlhPerGene = Maps.newTreeMap();
         for(String line : lines.subList(1, lines.size()))
         {
             String[] values = line.split(TSV_DELIM);
             String gene = values[geneIndex];
-            String mvlhString = values[mvlhIndex].substring(0, values[mvlhIndex].length() - 1);
-            double missedVariantLikelihood = Double.parseDouble(mvlhString) / 100D;
-            mvlhPerGene.put(gene, missedVariantLikelihood);
-        }
+            Double mvlh = Double.parseDouble(values[mvlhIndex]);
```
Comment (kduyvesteyn) [inline orange/src/main/java/com/hartwig/hmftools/orange/algo/OrangeAlgo.java:405]: Can be a `double` rather than `Double`

**Review by kduyvesteyn — APPROVED** [review]
Looks all good to me! Didn't test the actual MVLH table in the PDF though. 

Also, might be an idea to extract the MVLH logic out of OrangeAlgo and into a separate class. OrangeAlgo is mainly meant for tying together the various algo's, not really to execute anything non-trivial. Could also be for further improvement.

**Review by kzuberihmf — APPROVED** [review]
LGTM!

File: orange/src/main/java/com/hartwig/hmftools/orange/algo/OrangeAlgo.java (hunk 1 above)
Comment (DFKoetsier) [reply orange/src/main/java/com/hartwig/hmftools/orange/algo/OrangeAlgo.java]: Agreed

**Comment by DFKoetsier** [comment]
I have moved the MVLH logic from OrangeAlgo into a separate class

**Pull request merged**